contents/handbook/growth/marketing/newsletter.md

```diff
@@ -4,22 +4,24 @@ sidebar: Handbook
 showTitle: true
 ---
 
-We do not currently do email marketing, beyond product announcements via our newsletter, the PostHog Array. This is sent via Mailchimp. In the future, we will probably want to grow this channel. The Growth team use email for onboarding and other email flows, but this page covers marketing only. 
+We publish two newsletters:
 
-Speak to Charles, Yakko, or Charles for more information. 
+1. PostHog Array, a product announcement email published every four weeks
+2. PostHog News (working title), a marketing and content email published every two weeks
+
+Both emails are sent via Mailchimp. PostHog News (working title) is a work-in-progress newsletter which we plan to increase to a weekly schedule once we're confident we can produce it to a high standard at that higher frequency. PostHog News is primarily aimed at existing users as a means to improving engagement with our content and aid product discovery through sharing tutorials, tips and ideas, not user acquisition.
```

Comment:
Which metric of ours is PostHog News intended to help? I think knowing that will help us focus when in the future we're trying to decide what content to include or not include (e.g. could be Product Discoveries).

Reply:
Good question. I think influencing discoveries would be hard to attribute. I think the simplest metric is just 'pageviews'. If our users are reading our content, be it tutorials, they'll either be learning something new or engaging more with us as a brand. CTR as measured in Mailchimp (do we have a Mailchimp plug-in?) would be a good quality metric, as wouuld open rates etc.

Reply:
Yeah, that all makes sense. Discoveries feels like a sensible option.

Reply:
(Good discussion)

I'd have assumed more word of mouth growth would be our goal here.

However, we view this as an expansion of usage exercise?

If so feels like we need way more tutorial stuff in the calendar...

Reply:
I think it sits in both, really. My view is we can deliver a newsletter that our users enjoy and find useful, it's another means of making them into advocates for us.

Reply:
> However, we view this as an expansion of usage exercise?

I agree with that - quality content to existing users/newsletter subscribers (who may not be users) should increase their in-org usage, either through recipients spending more time in the product or adding more users within their org. 

@andyvan-ph correct me if I'm wrong, but I believe the intention is to have every newsletter include a tutorial (starting with something every 2 weeks until we're confident we can increase frequency).

Reply:
Yes, that's correct. I [have a draft](https://docs.google.com/document/d/1hknRyf5Zb6cLuSyEpnawVw6NdRBnva6bQcL3vo37cZw/edit) for comments, though maybe I should make this a PR? Let me know if you'd prefer that showing that.

Basic idea is:

- Newsletter every two weeks
- Every newsletter will feature a blog post and a tutorial
- Some curated content from elsehwhere that we think is interesting and relevant
- A fun cartoon from Lottie despiting a movie/TV show etc. to add some character

But still playing around with it and could really do with some initial impressions.